setup.cfg

```diff
@@ -7,9 +7,8 @@ license_file = LICENSE
 [flake8]
 max-line-length = 100
 max-complexity = 10
-
-[pep8]
```

Comment:
Do other tools that inspect the `[pep8]` section all have a fallback to the `[flake8]` section?

Reply:
"pep8" was changed to "pycodestyle" several years ago at the BDFL's request.  flake8 is a superset of pycodestyle so it does everything pycodestyle did and much more besides.

Reply:
Even then, such cleanups should go in a different commit. Getting flake8 to run in CI has little to do with removing unused bits of config.